test/functional/ui/screen.lua

```diff
@@ -496,8 +501,8 @@ function pprint_attrs(attrs)
     for f, v in pairs(attrs) do
       local desc = tostring(v)
       if f == "foreground" or f == "background" then
-        if colornames[v] ~= nil then
-          desc = "Screen.colors."..colornames[v]
+        if Screen._colornames[v] ~= nil then
+          desc = "Screen.colors."..Screen._colornames[v]
```

Comment:
`Screen.colors().` for consistency here, but tell me, why should `Screen.colors` be a function? It is static data that doesn't depend on the state of nvim.